x/upgrade/types/storeloader.go

```diff
@@ -10,9 +10,9 @@ import (
 // pattern. This is useful for custom upgrade loading logic.
 func UpgradeStoreLoader(upgradeHeight int64, storeUpgrades *store.StoreUpgrades) baseapp.StoreLoader {
 	return func(ms sdk.CommitMultiStore) error {
-		if upgradeHeight == ms.LastCommitID().Version {
+		if upgradeHeight == ms.LastCommitID().Version+1 {
```

Comment:
We stop the node process on `beginblock(upgradeHeight)`, the block is not commited, so it should be `LastCommitID().Version+1`.